src/Components/Admin/EnvironmentEditor/sceneCard.js

```diff
@@ -87,15 +87,32 @@ export default function SceneCard({
                                 handleEditClick(scene.id);
                             }}
                         >
-                            Edit
+                            Edit Metadata
+                        </MenuItem>
+                        <MenuItem>
+                            <a
+                                href={
+                                    process.env.REACT_APP_ADMIN_BACKEND_URL +
+                                    "/admin/scene/" +
+                                    scene.id
+                                }
+                                target="_blank"
+                                rel="noreferrer"
+                                style={{
+                                    textDecoration: "none",
+                                    color: "#000",
```

Comment:
I would've thought textDecoration resets color too? But I assume you tried without this so it's okay to keep.

Reply:
Yea tried without the color attribute, it was still purple, hence it being here